index.d.ts

```diff
@@ -0,0 +1,141 @@
+import cliBoxes, {BoxStyle} from 'cli-boxes';
```

Comment:
Can you re-export the `BoxStyle` type?

And also use this in the type tests instead of from `cli-boxes`.

Reply:
This is basically what I was doing with `CustomBorderStyle`. Should I remove it?

Reply:
Ah, sorry, missed that. No, just change the type test to use `CustomBorderStyle` instead of `BoxStyle`.

Reply:
And what about the TypeScript "bug"? It doesn't appear to be getting much traction. I can change it so that the values work with current tooling, maybe with a TODO referencing the issue.